app/assets/stylesheets/layout.scss

```
@@ -1453,6 +1454,16 @@ table {
   }
 }
 
+.button.button-oidc {
+  @include has-fa-icon(openid, brands);
+  background: #fafafa;
+  border-left: 3px solid #fdf9f1;
```

I think there's a typo here, right?

```suggestion
  background: #fdf9f1;
  border-left: 3px solid #f7931e;
```